app/views/content/train-to-be-a-teacher/get-school-experience.md

```diff
@@ -19,7 +19,7 @@ calls_to_action:
         link_target: "https://schoolexperience.education.gov.uk/"
 promo_content:
     - content/train-to-be-a-teacher/promos/mailing-list-promo
-navigation: 5.50
+navigation: 20.20
```

Comment:
I think you might need to be a bit careful here that you don't have conflicting numbers over the two PRs (quals and GSE)?